client/descriptions.go

```diff
@@ -29,6 +29,9 @@ type CollectionDescription struct {
 
 	// Schema contains the data type information that this Collection uses.
 	Schema SchemaDescription
+
+	// Indexes contains the indexes that this Collection has.
+	Indexes []IndexDescription
```

Comment:
originally `[]IndexDescription` was separated from `CollectionDescription` but @jsimnz insisted to include it here.
What do you mean by "disable"?
I'm not even sure what would be the best way to test it because the observable behaviour is not quite observable.

Reply:
The location here is correct IMO.  By disable, I mean make it return an error if such a modification is attempted - this is already done for some stuff, for example `TestSchemaUpdatesRemoveFieldErrors` and `TestSchemaUpdatesAddSimpleErrorsAddingSchema`.

You can use these tests as examples as to how to test it.